dataflow/src/main/java/org/checkerframework/dataflow/cfg/AbstractCFGVisualizer.java

```diff
@@ -90,7 +89,7 @@ protected String visualizeGraph(
      */
     protected String visualizeGraphWithoutHeaderAndFooter(
             ControlFlowGraph cfg, Block entry, @Nullable Analysis<V, S, T> analysis) {
-        Set<Block> visited = new HashSet<>();
+        Set<Block> visited = new LinkedHashSet<>();
```

Comment:
This change has been moved to typetools: https://github.com/typetools/checker-framework/pull/3394